### ISSUE TITLE
Badge's async boolean flag conflicts with async keyword in Python >3.6

### ISSUE THREAD

**Issue by piraka9011** [pinax/pinax-badges#36]
**Describe the bug**
In badges/base.py, the badge class has a boolean variable name `async`. This conflicts with Python's `async` keyword

**Reproduce behavior**

Run any `manage.py` task. Ex.

```sh
python manage.py makemigrations
```

**Expected behavior**

Migrations should run successfully.

**Screenshots**

<details><summary>Full stack trace:</summary>
<p>

```sh
Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pycharm/django_manage.py", line 52, in <module>
    run_command()
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pycharm/django_manage.py", line 46, in run_command
    run_module(manage_file, None, '__main__', True)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/allabana/tarteel_ws/tarteel-api/manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/base.py", line 361, in execute
    self.check()
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/urls/resolvers.py", line 399, in check
    for pattern in self.url_patterns:
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/urls/resolvers.py", line 584, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/utils/functional.py", line 80, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/urls/resolvers.py", line 577, in urlconf_module
    return import_module(self.urlconf_name)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/allabana/tarteel_ws/tarteel-api/tarteel/urls.py", line 33, in <module>
    path('v1/badges/', include("pinax.badges.urls", namespace="pinax_badges")),
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/pinax/badges/urls.py", line 3, in <module>
    from .views import badge_detail, badge_list
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/pinax/badges/views.py", line 7, in <module>
    from .registry import badges
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/pinax/badges/registry.py", line 1, in <module>
    from .base import Badge
  File "/Users/allabana/.virtualenvs/te-api/lib/python3.7/site-packages/pinax/badges/base.py", line 18
    async = False
```

</p>
</details>

**Desktop**
 - OS: Mac OS 10.15
 - Browser: N/A
 - Version: 2.0.0

**Additional context**

This can be resolved by refactoring the `async` variable to another name. If this is fine with the maintainers, I can open a PR.

**Comment by KatherineMichel** [comment]
As far as I know, we fixed this a while back and published as 2.0.3. Do we need to take further action, @piraka9011?
https://github.com/pinax/pinax-badges/pull/30/files
https://pypi.org/project/pinax-badges/#history

**Comment by piraka9011** [comment]
@KatherineMichel I did not notice the minor version bump. I will close this issue, thanks!
